File: Python Programming/problems on numbericals/problem7.py
'''
7.Write a program which accept number from user and print even factors of that number. 
		
	Author: Milind Nalawade
	
	Date: 19/06/2021

	language: Python 3.9.0
'''	

#=============
#Entry point function
#=============

def main():
	no=int(input("Enter your number : "))
	i=2
	j=100
	
	while j!=0:
		
		if i%no==0:
			print(i,end=" ")
		
		i=i+2
		j=j-1	
		
#===========
#starter function
#===========
if __name__=="__main__":
	main()

File: C Programming/problem on numbericals/problem7.c
/*
   7.Write a program which accept number from user and print even factors of that number. 
	
	Author: Milind Nalawade
	
	Date: 19/06/2021

	language: C programming
*/

#include<stdio.h>

//Entry point function

int main()
{
	int a=0,i=2,j=0;
	printf("Enter your number : ");
	scanf("%d",&a);
	j=100;
	while(j)
	{
		if(i%a==0)
	{	
		printf("%d ",i);
		
	}
	i=i+2;	
		j--;
	}
	return 0;
}

